sesman/chansrv/chansrv_config.c

```diff
@@ -211,6 +244,7 @@ new_config(void)
         cfg->fuse_mount_name = fuse_mount_name;
         cfg->file_umask = DEFAULT_FILE_UMASK;
         cfg->use_nautilus3_flist_format = DEFAULT_USE_NAUTILUS3_FLIST_FORMAT;
+        cfg->restrict_inbound_clipboard = DEFAULT_RESTRICT_INBOUND_CLIPBOARD;
```

Comment:
Purely stylistically, I'd suggest keeping `cfg->restrict_outbound_clipboard` and `cfg->restrict_inbound_clipboard` together, as they're doing similar jobs. This applies to the header file too.

Reply:
Fixed.